src/components/layout/BookingCancellation/BookingCancel/BookingCancel.jsx

```diff
@@ -21,9 +21,11 @@ const BookingCancel = ({ isEvent, booking }) => {
       </div>
 
       <p className="mt40">
-        <span className="is-block">
-          {`${price} vont être recrédités sur ton pass.`}
-        </span>
+        {isWalletValid === true && (
```

Comment:
Ne peut-on pas juste écrire : 
```suggestion
        {isWalletValid && (
```